packages/scoped-custom-element-registry/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed form-associated custom element definitions (`static formAssociated = true`) when used with the polyfill.
 - Fixed patched callback names in form-associated custom element support.
 
+## [0.0.4] - 2022-26-01
+
+- Lowercasing of attributes. Fixes #483
```

Comment:
```suggestion
- Fixed handling of mixed-case attributes. Fixes #483
```

Reply:
Note we create the release sections when we release, so for now it should stay under Unreleased.